examples/Expr.hs

```diff
@@ -81,6 +81,18 @@ type Forest a = [Tree a]
 
 makeBaseFunctor ''Tree
 
+data Rose f a = Rose [f (Rose f a)]
+data Qose f a = Qose [f (Qose f a)]
+
+makeBaseFunctor [d|
+  instance Functor f => Recursive (Rose f a)
+  instance Functor f => Recursive (Qose f a)
+  |]
```

Comment:
`[d| ... |]` produces a `Q [Dec]` even when there's only one declaration in the block, so I don't think the example needs two lines. In fact, supporting more than one declaration is also an accidental feature!

Reply:
I think supporting more than one declaration is a friendly feature worth preserving.